package.json

```diff
@@ -58,52 +78,52 @@
     "electron-devtools-installer": "^2.2.4",
     "electron-settings": "^3.2.0",
     "electron-window-state": "^5.0.3",
-    "notistack": "^0.5.1",
+    "notistack": "^0.8.4",
     "prop-types": "^15.7.2",
-    "react": "16.8.5",
-    "react-color": "^2.14.1",
-    "react-dom": "16.8.5",
+    "react": "16.8.6",
+    "react-color": "^2.17.3",
+    "react-dom": "16.8.6",
     "react-localization": "^1.0.13",
-    "source-map-support": "^0.5.11",
+    "source-map-support": "^0.5.12",
     "typeface-roboto": "^0.0.54",
     "typeface-source-code-pro": "^0.0.71",
     "usb": "^1.5.0"
   },
   "devDependencies": {
-    "@babel/cli": "^7.1.0",
-    "@babel/core": "7.4.0",
-    "@babel/plugin-proposal-class-properties": "^7.4.0",
+    "@babel/cli": "^7.4.4",
+    "@babel/core": "7.4.4",
+    "@babel/plugin-proposal-class-properties": "^7.4.4",
     "@babel/plugin-syntax-dynamic-import": "^7.0.0",
-    "@babel/preset-env": "^7.4.2",
+    "@babel/preset-env": "^7.4.4",
     "@babel/preset-react": "^7.0.0",
     "babel-core": "^7.0.0-bridge.0",
     "babel-eslint": "^10.0.1",
-    "babel-jest": "^24.5.0",
-    "babel-loader": "^8.0.2",
+    "babel-jest": "^24.8.0",
+    "babel-loader": "^8.0.6",
     "electron": "4.1.1",
-    "electron-builder": "^20.39.0",
-    "electron-webpack": "^2.0.1",
-    "electron-webpack-eslint": "^4.0.1",
+    "electron-builder": "^20.40.2",
+    "electron-webpack": "^2.6.2",
```

Comment:
This has been causing issues, see #365. Until we verified that these updates work with a prod build, not just with `yarn run start`, I'd rather not update any dependencies, nor `yarn.lock`.

Reply:
Ahh, didn't catch that part in that thread, got it.